website/docs/cloud-docs/api-docs/configuration-versions.mdx

```diff
@@ -434,3 +436,90 @@ The GET endpoints above can optionally return related resources, if requested wi
 | -------------------- | ------------------------------------------------- |
 | `ingress_attributes` | The commit information used in the configuration. |
 | `run`                | The run created by the configuration.             |
+
+## Soft-Delete a Configuration Version
+
+`POST /api/v2/configuration-versions/:external_id/actions/soft_delete_backing_data`
+
+| Parameter                   | Description                                    |
+| --------------------------- | ---------------------------------------------- |
+| `external_id`  | The external ID of the configuration version to soft-delete. |
+
+This endpoint notifies Terraform Enterprise to soft-delete backing files associated with this configuration version. This endpoint can only soft-delete configuration versions that were created with the API or CLI, are in an `uploaded` [state](#configuration-version-states), and are not the current configuration version. Otherwise, calling this endpoint will result in an error.
+
+| Status  | Response                  | Reason(s)                                                                                                                                     |
+| ------- | ------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------- |
+| [200][] | none                      | Successfully initiated the soft-delete process.                                                                                                   |
+| [404][] | [JSON API error object][] | Configuration version was not found or user not authorized.                                                                                   |
```

Comment:
Since it's called out above (and can happen!) it might be worth adding a row for the invalid state case.

Reply:
Great suggestion! I've updated this as well ✔️